src/Native/.clang-format

```diff
@@ -0,0 +1,16 @@
+---
+Language:        Cpp
+# BasedOnStyle:  LLVM
+AlignEscapedNewlinesLeft: false
+AllowShortFunctionsOnASingleLine: false
+AlwaysBreakTemplateDeclarations: true
+BinPackParameters: false
+BreakBeforeBraces: Allman
+ColumnLimit:     150
```

Comment:
I picked 150 chars since that was big enough to fit on the most common screen resolution (1366x768) but okay changing it if there is pushback against this